src/index.css

```diff
@@ -147,7 +147,8 @@ a {
 }
 
 .header-img {
-  height: 300px;
+  height: 340.66px;
```

Comment:
I think you have height and width flipped. Using the current vaxx.nz deployment and inspecting the values indicates that height should be `300px` and width `340.66px`.